File: app_manager/appmanager/src/main/java/app/editors/manager/mvp/models/request/RequestBatchOperation.java
package app.editors.manager.mvp.models.request;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class RequestBatchOperation extends RequestBatchBase {

    @SerializedName("destFolderId")
    @Expose
    private String destFolderId;

    @SerializedName("conflictResolveType")
    @Expose
    private int conflictResolveType;

    public String getDestFolderId() {
        return destFolderId;
    }

    public void setDestFolderId(String destFolderId) {
        this.destFolderId = destFolderId;
    }

    public int getConflictResolveType() {
        return conflictResolveType;
    }

    public void setConflictResolveType(int conflictResolveType) {
        this.conflictResolveType = conflictResolveType;
    }

}

File: app_manager/core/src/main/java/app/documents/core/network/manager/models/request/RequestBatchOperation.kt
package app.documents.core.network.manager.models.request

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

data class RequestBatchOperation(
    @SerializedName("destFolderId")
    @Expose
    var destFolderId: String = "",

    @SerializedName("conflictResolveType")
    @Expose
    var conflictResolveType: Int = 2,

    @SerializedName("content")
    @Expose
    var content: Boolean = true,

) : RequestBatchBase()